flow/connectors/postgres/postgres.go

```diff
@@ -79,6 +81,11 @@ func NewPostgresConnector(ctx context.Context, pgConfig *protos.PostgresConfig)
 		metadataSchema = *pgConfig.MetadataSchema
 	}
 
+	vigil, err := evervigil.NewVigil()
```

Comment:
should `NewVigil` take ctx given it's calling `Context.Background()`?

Reply:
No, if you want that level of isolation you'll want the monitor to be a separate container